notifier/tests/test_commands.py

```diff
@@ -21,3 +26,31 @@ class CommandsTestCase(TestCase):
                        BROKER_BACKEND='memory',)
     def test_forums_digest(self):
         pass
+
+    def test_forums_digests_flagged_dry(self):
+        """
+        Test that command with dry run option works.
+        """
+        test_threads = (
+            {
+                'course_id': 'org/course/run',
+                'commentable_id': 'test_commentable',
+                'comment_thread_id': i,
+            }
+            for i in xrange(1, 6)
+        )
+        test_moderators = (usern(i) for i in xrange(10))
+        with patch('notifier.management.commands.forums_digest_flagged.get_flagged_threads', return_value=test_threads), \
+             patch('notifier.management.commands.forums_digest_flagged.get_moderators', return_value=test_moderators):
+            out = sys.stdout
+            sys.stdout = StringIO()
+            management.call_command('forums_digest_flagged',
+                is_dry_run=True
+            )
+            output = sys.stdout.getvalue()
+            for i in xrange(10):
+                self.assertIn("To: {}".format(usern(i)['email']), output)
+            self.assertIn("org/course/run", output)
```

Comment:
At the least, let's make this a local variable, if not a global, so we can reference it elsewhere without typing the string literal.